packages/bpk-component-popover/readme.md

```diff
@@ -42,23 +42,31 @@ class App extends Component {
 
   render() {
     return (
-      <BpkPopover
-        id="my-popover"
-        target={
-          <BpkButton onClick={this.openPopover}>Open</BpkButton>
-        }
-        onClose={this.closePopover}
-        isOpen={this.state.isOpen}
-        label="My popover"
-        closeButtonText="Close"
-      >
-        <BpkText>My popover content</BpkText>
-      </BpkPopover>
+      <div id="popover-container">
+        <BpkPopover
+          id="my-popover"
+          target={
+            <BpkButton onClick={this.openPopover}>Open</BpkButton>
+          }
+          onClose={this.closePopover}
+          isOpen={this.state.isOpen}
+          label="My popover"
+          closeButtonText="Close"
+          renderTarget={() =>
+            document.getElementById('popover-container')
+          }
+        >
+          <BpkText>My popover content</BpkText>
+        </BpkPopover>
+      </div>
     );
   }
 }
 ```
 
+> **Theming:** In order to theme the popover, a `renderTarget` needs to be supplied as a function which returns a DOM node
```

Comment:
@matthewdavidson should we point out the support for refs? technically we support both 🤔

Reply:
Erm we could ... I guess hugo probably wants that too. I guess also we'd need to update the modal docs too 🙈

Reply:
I can put it in my PR, is not a major issue, something like
```
> **Theming:** In order to theme the popover, a `renderTarget` needs to be supplied as either a function which returns a DOM node or a [ref](https://reactjs.org/docs/refs-and-the-dom.html) in the scope of a `BpkThemeProvider`.
```

Reply:
ya!